File: inject-java/src/test/groovy/io/micronaut/inject/configproperties/Pojo.java
package io.micronaut.inject.configproperties;

import io.micronaut.core.annotation.Introspected;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;

@Introspected
public class Pojo {

    @Email(message = "Email should be valid")
    private String email;

    @NotBlank
    private String name;

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: inject-kotlin/src/test/kotlin/io/micronaut/kotlin/processing/inject/configproperties/Pojo.kt
package io.micronaut.kotlin.processing.inject.configproperties

import io.micronaut.core.annotation.Introspected
import javax.validation.constraints.Email
import javax.validation.constraints.NotBlank

@Introspected
class Pojo {

    @Email(message = "Email should be valid")
    var email: String? = null

    @NotBlank
    var name: String? = null
}

